tests/e2e/uninstall.go

```diff
@@ -0,0 +1,194 @@
+		requireRemoveKoperatorCRDs(kubectlOptions)
+	})
+}
+
+// requireUninstallingKoperatorHelmChart uninstall Koperator Helm chart
+// and checks the success of that operation.
+func requireUninstallingKoperatorHelmChart(kubectlOptions k8s.KubectlOptions) {
+	It("Uninstalling Koperator Helm chart", func() {
+		uninstallHelmChartIfExists(kubectlOptions, koperatorLocalHelmDescriptor.ReleaseName, true)
+		By("Verifying Koperator helm chart resources cleanup")
+		k8sResourceKinds, err := listK8sResourceKinds(kubectlOptions, "")
+		Expect(err).ShouldNot(HaveOccurred())
+
+		koperatorAvailableResourceKinds := stringSlicesInstersect(koperatorCRDs(), k8sResourceKinds)
+		koperatorAvailableResourceKinds = append(koperatorAvailableResourceKinds, basicK8sResourceKinds()...)
+
+		remainedResources, err := getK8sResources(kubectlOptions,
+			koperatorAvailableResourceKinds,
+			fmt.Sprintf(managedByHelmLabelTemplate, koperatorLocalHelmDescriptor.ReleaseName),
+			"",
+			kubectlArgGoTemplateKindNameNamespace,
+			"--all-namespaces")
+
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(remainedResources).Should(BeEmpty())
+	})
+}
+
+// requireUninstallingZookeeperOperator uninstall Zookeeper-operator Helm chart
+// and remove CRDs.
+func requireUninstallingZookeeperOperator(kubectlOptions k8s.KubectlOptions) {
+	When("Uninstalling zookeeper-operator", func() {
+		requireUninstallingZookeeperOperatorHelmChart(kubectlOptions)
+		requireRemoveZookeeperOperatorCRDs(kubectlOptions)
+	})
+}
+
+// requireUninstallingZookeeperOperatorHelmChart uninstall Zookeeper-operator Helm chart
+// and checks the success of that operation.
+func requireUninstallingZookeeperOperatorHelmChart(kubectlOptions k8s.KubectlOptions) {
+	It("Uninstalling zookeeper-operator Helm chart", func() {
+		uninstallHelmChartIfExists(kubectlOptions, zookeeperOperatorHelmDescriptor.ReleaseName, true)
+		By("Verifying Zookeeper-operator helm chart resources cleanup")
+
+		k8sResourceKinds, err := listK8sResourceKinds(kubectlOptions, "")
+		Expect(err).ShouldNot(HaveOccurred())
+
+		zookeeperAvailableResourceKinds := stringSlicesInstersect(zookeeperCRDs(), k8sResourceKinds)
+		zookeeperAvailableResourceKinds = append(zookeeperAvailableResourceKinds, basicK8sResourceKinds()...)
+
+		remainedResources, err := getK8sResources(kubectlOptions,
+			zookeeperAvailableResourceKinds,
+			fmt.Sprintf(managedByHelmLabelTemplate, zookeeperOperatorHelmDescriptor.ReleaseName),
+			"",
+			kubectlArgGoTemplateKindNameNamespace,
+			"--all-namespaces")
+		Expect(err).ShouldNot(HaveOccurred())
+
+		Expect(remainedResources).Should(BeEmpty())
+	})
+}
+
+// requireRemoveZookeeperOperatorCRDs deletes the zookeeper-operator CRDs
+func requireRemoveZookeeperOperatorCRDs(kubectlOptions k8s.KubectlOptions) {
+	It("Removing zookeeper-operator CRDs", func() {
+		for _, crd := range zookeeperCRDs() {
+			deleteK8sResourceNoErrNotFound(kubectlOptions, defaultDeletionTimeout, crdKind, crd)
```

Comment:
**Required**: We **MUST** check potential errors here.

```suggestion
			err := deleteK8sResourceNoErrNotFound(kubectlOptions, defaultDeletionTimeout, crdKind, crd)
			
			Expect(err).NotTo(HaveOccurred())
```

Reply:
fixed: [3efc5f9](https://github.com/banzaicloud/koperator/pull/987/commits/3efc5f92e73093a477784196d6bcf64ffcc0a417)